janitor/functions/drop_duplicate_columns.py

```diff
@@ -51,7 +55,7 @@ def drop_duplicate_columns(
     removed_col_idx = col_indexes[nth_index]
     # get the column indexes without column that is being removed
     filtered_cols = [
-        c_i for c_i, c_v in enumerate(cols) if c_i != removed_col_idx
+        c_i for c_i, c_v in enumerate(df.columns) if c_i != removed_col_idx
```

Comment:
This implementation looks a little peculiar to me. Though nothing wrong logically, but since `c_v` is not used, it appears to me that enumerate is not necessary here.

Something like
```python
filtered_cols = [idx for idx in range(df.shape[1]) if idx != removed_col_idx]
```
seems like it would work as well?

Reply:
Good point. I think the implementation could be simplified further too, by using `df.drop` instead. Let me try that out.

Reply:
Never mind, I was wrong :smile:. Using `df.drop` makes things a bit more difficult!

Reply:
@thatlittleboy on second thought, I think iterating over `df.columns` is a bit more explicit than doing `df.shape[1]`. I've replaced the loop variable `c_v` with an underscore `_` instead, which together with `enumerate(df.columns)` should help make clear that we're only interested in the index of each column name.

Reply:
Yep, understand. That works too!